requirements.txt

```diff
@@ -1,10 +1,9 @@
 alembic==1.0.11
 Click==7.0
 dominate==2.4.0
-entrypoints==0.3
-flake8==3.7.8
 Flask==1.1.1
 Flask-Bootstrap==3.3.7.1
```

Comment:
maybe delete bootstap3?

Reply:
Flask-Bootstrap4 just adds bootstrap4 styles to Flask-Bootstrap module. The app won't be work without main Flask-Bootstrap module.